packages/plugin-client-drizzle/src/session.ts

```diff
@@ -78,8 +78,17 @@ export class XataPreparedQuery<T extends PreparedQueryConfig> extends PreparedQu
       );
 
     if (warning) console.warn(warning);
-
-    const rows = records.map((record) => fields.map((field) => record[field.path.join('.')]));
+    const internalColumnNames = ['xata.version', 'xata.createdAt', 'xata.updatedAt', 'xata.deletedAt'];
```

Comment:
This should maybe go somewhere else outside of this file?

Reply:
These are changing right now with SQL. I think we can leave it as is and fix it for v1